src/tactic/squeeze.lean

```diff
@@ -64,6 +64,12 @@ def squeeze_loc_attr_carrier := ()
 
 run_cmd squeeze_loc_attr.set ``squeeze_loc_attr_carrier none tt
 
+/-- Format a list of arguments for use with `simp` and friends. This omits the
+list entirely if it is empty. -/
+meta def render_simp_arg_list : list simp_arg_type → tactic format
+| [] := pure ""
+| args := (++) " " <$> to_line_wrap_format <$> args.mmap pp
```

Comment:
I probably misunderstand this, but doesn't this put the space after the list, and you want it before the list?

Reply:
Lean doesn't have left sections, so you write `\lam x, " " ++ x` as `(++) " "` and `\lam x, x ++ " "` as `(++ " ")`.

Reply:
Oh, you're right. I didn't realize the distinction of these similar-looking functions. I'm going to remember that!